english/watch-sunset/watch-sunset.md

```diff
@@ -0,0 +1,3 @@
+
+As you watch the sun dip into the water, you think about all the things that have brought you to this point, like the plane, and the rental car. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.